src/modules/conversations.ts

```diff
@@ -35,6 +35,20 @@ class ConversationsApi extends BaseApi {
     showInteractionDetails(conversationId: string) {
         super.sendMsgToPc('showInteractionDetails', {conversationId: conversationId});
     }
+
+    /**
+     * Send a message to be filled into the interaction message box for the agent to review and send.
+     * This function works specifically with a bound interaction when both the interaction and calling app
+     * are visible, it is not intended (and will not work) for situations where the interaction is not active.
```

Comment:
Can you add a sentence to the effect of,

"Practically, this means that you should only call this method in response to user interaction to ensure the agent is aware of the impending text insertion and so their existing draft state is not unexpectedly altered."